src/pages/api/register.ts

```diff
@@ -0,0 +1,95 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import Auth from '@aws-amplify/auth';
+import { getDomain, redirectTo, redirectToError, setCookieOnResponseObject } from './apiUtils';
+import { USER_COOKIE } from '../../constants';
+import { InputCheck } from '../register';
+import { getServicesByNocCode } from '../../data/auroradb';
+
+const checkEmailValid = (email: string) => {
```

Comment:
Do we deal with whitespace by removing it at some point? And do we sanitise the emails incase of Alt Caps somewhere? I remember we said we wouldn't do this when we show their email on the page after they create it, but I think we said in the backend we want to have it all in lower-case.

Reply:
We will need to trim potentially but cognito will handle the casing